content/en/serverless/installation/nodejs.md

```diff
@@ -59,6 +59,39 @@ To install and configure the Datadog Serverless Plugin, follow these steps:
     ```
     More information on the Datadog Forwarder ARN or installation can be found [here][2]. For additional settings, see the [plugin documentation][1].
 
+4. (Optional - only applies if you're using Webpack)
+
+    `dd-trace` is known to be not compatible with webpack due to the use of conditional import and other issues. If using webpack, make sure to mark `datadog-lambda-js` and `dd-trace` as [externals](https://webpack.js.org/configuration/externals/) for webpack, so webpack knows these dependencies will be available in the runtime. You should also remove `datadog-lambda-js` and `dd-trace` from `package.json` and the build process to ensure you're using the versions provided by the Datadog Lambda Layer.
+
+    #### serverless-webpack
```

Comment:
don't think `#### serverless-webpack` is needed here, or at least don't need to be a title, could be simply bolded if needed.